theory/special stars/black holes.md

```diff
@@ -7,3 +7,52 @@ nav_order: 3
+
+### Types of Black Holes
+
+[Black holes](#black-holes) come in various sizes and are categorized by their mass:
+
+1. Stellar Black Holes: These form when massive stars collapse and typically range from 3 to 20 times the mass of the Sun. They are often part of [binary systems](binary%20stars.html), pulling material from companion stars and emitting powerful X-rays.
+
+2. Supermassive Black Holes: Found at the centers of most galaxies, including the Milky Way, these black holes have masses ranging from millions to billions of times that of the Sun. Their formation is still not fully understood, but they are critical to galaxy formation.
+
+3. Intermediate-Mass Black Holes: These rare black holes are thought to be formed by the merging of smaller black holes or the collapse of very large stars, with masses between stellar and supermassive black holes.
+
+4. Primordial Black Holes: Hypothetical black holes that might have formed in the early universe shortly after the Big Bang, potentially providing insights into dark matter.
+
+### Effects on Surroundings
+
+Though invisible, [black holes](#black-holes) reveal themselves through their effects on nearby matter and light. Material drawn towards a black hole forms an accretion disk, emitting intense radiation as it heats up. This radiation, especially in the X-ray spectrum, allows astronomers to detect black holes.
+
+1. Gravitational Lensing: The warping of space-time around a black hole can bend light, distorting or magnifying the image of distant objects behind it. This allows black holes to act as cosmic lenses.
+
+2. Time Dilation: As predicted by Einstein’s theory of general relativity, time slows down near a black hole’s event horizon. An observer watching an object fall into a black hole would see it slow down and freeze as it approaches the event horizon.
+
+3. [Gravitational Waves](../cosmology/gravitational%20waves.html): When two black holes merge, they produce ripples in space-time known as [gravitational waves](../cosmology/gravitational%20waves.html). These waves have been detected by observatories like LIGO, confirming Einstein’s predictions and providing new insights into black hole properties.
+
+### Black Hole Information Paradox
+
+One of the biggest unresolved puzzles in physics is the [black hole](#black-holes) information paradox. According to quantum mechanics, information cannot be destroyed, yet anything that falls into a black hole seems to be lost forever. This conflict with general relativity has sparked significant debate. Some theories suggest information may be encoded at the event horizon or could escape through Hawking radiation, a theoretical process by which black holes gradually lose mass.
```

Comment:
Please hyperlink the article on [black hole information paradox](https://astroclubiitk.in/2020-09-02-blackhole-paradox/) as a callout here